sessionctx/variable/featuretag/distributereorg/default.go

```diff
@@ -17,4 +17,4 @@
 package distributereorg
 
 // TiDBEnableDistributeReorg is a feature tag
-const TiDBEnableDistributeReorg bool = false
+const TiDBEnableDistributeReorg bool = true
```

Comment:
We will default enable `DistributeReorg`?

Reply:
No. It's a mistake.